### PR TITLE
[zh]Update content/zh/docs/reference/kubectl/cheatsheet.md

### PR DESCRIPTION
**Pull request by zyy19981018**

### CODE REVIEW

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -69,6 +70,19 @@ echo "[[ $commands[kubectl] ]] && source <(kubectl completion zsh)" >> ~/.zshrc
 source <(kubectl completion zsh)  # 在 zsh 中设置当前 shell 的自动补全
 echo "[[ $commands[kubectl] ]] && source <(kubectl completion zsh)" >> ~/.zshrc # 在您的 zsh shell 中永久的添加自动补全
 ```
+### A Note on --all-namespaces
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:73]: 这一行要包含在 76 行之前，注释掉。

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -429,14 +453,16 @@ kubectl diff -f ./my-manifest.yaml
...
+kubectl get nodes -o json | jq -c 'path|join(".")'
 
 # 生成一个句点分隔的树，其中包含为pod等返回的所有键
-kubectl get pods -o json | jq -c 'path(..)|[.[]|tostring]|join(".")'
+kubectl get pods -o json | jq -c 'path|join(".")'
 
 # 假设你的 Pods 有默认的容器和默认的名字空间，并且支持 'env' 命令，可以使用以下脚本为所有 Pods 生成 ENV 变量。
 # 该脚本也可用于在所有的 Pods 里运行任何受支持的命令，而不仅仅是 'env'。 
 for pod in $(kubectl get po --output=jsonpath={.items..metadata.name}); do echo $pod && kubectl exec -it $pod -- env; done
+# 获取部署的状态子资源
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:464]:
```suggestion
# 获取 Deployment 的 status 子资源
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 3]
```diff
@@ -529,6 +558,9 @@ kubectl patch deployment valid-deployment  --type json   -p='[{"op": "remove", "
 
 # 在带位置数组中添加元素
 kubectl patch sa default --type='json' -p='[{"op": "add", "path": "/secrets/1", "value": {"name": "whatever" } }]'
+
+# 通过修复其规模子资源来更新部署的副本数
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:562]:
```suggestion
# 通过修改 Deployment 的 scale 子资源来更新其副本数
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 4]
```diff
@@ -576,16 +608,18 @@ kubectl scale --replicas=5 rc/foo rc/bar rc/baz                   # 伸缩多个
...
+kubectl delete pod,service baz foo                                # Delete pods and services with same names "baz" and "foo"
+kubectl delete pods,services -l name=myLabel                      # Delete pods and services with label name=myLabel
+kubectl -n my-ns delete pod,svc --all                             # Delete all pods and services in namespace my-ns,
 # Delete all pods matching the awk pattern1 or pattern2
 kubectl get pods  -n mynamespace --no-headers=true | awk '/pattern1|pattern2/{print $1}' | xargs  kubectl delete -n mynamespace pod
 ```
 -->
 ```bash
 kubectl delete -f ./pod.json                                              # 删除在 pod.json 中指定的类型和名称的 Pod
+kubectl delete pod unwanted --now                                         # 删除没有宽限期的 pod
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:622]:
```suggestion
kubectl delete pod unwanted --now                                         # 删除 Pod，不受宽限期约束
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 5]
```diff
@@ -648,6 +681,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+kubectl exec -n my-namespace my-pod -- tar cf - /tmp/foo | tar xf - -C /tmp/bar    # Copy /tmp/foo from a remote pod to /tmp/bar locally
+```
+-->
+```bash
+kubectl cp /tmp/foo_dir my-pod:/tmp/bar_dir            # 将 /tmp/foo_dir 本地目录复制到当前命名空间中远程 pod 中的 /tmp/bar_dir
+kubectl cp /tmp/foo my-pod:/tmp/bar -c my-container    # 将 /tmp/foo 本地文件复制到特定容器中远程 pod 中的 /tmp/bar
+kubectl cp /tmp/foo my-namespace/my-pod:/tmp/bar       # 将 /tmp/foo 本地文件复制到命名空间 my-namespace 中远程 pod 中的 /tmp/bar
+kubectl cp my-namespace/my-pod:/tmp/foo /tmp/bar       # 将 /tmp/foo 从远程 pod 复制到本地 /tmp/bar
+```
+{{<注>}}
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:712]: `{{< note >}}` 是短代码，不要翻译。

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 6]
```diff
@@ -648,6 +681,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+{{< /note >}}
+
+```bash
+tar cf - /tmp/foo | kubectl exec -i -n my-namespace my-pod -- tar xf - -C /tmp/bar           # Copy /tmp/foo local file to /tmp/bar in a remote pod in namespace my-namespace
+kubectl exec -n my-namespace my-pod -- tar cf - /tmp/foo | tar xf - -C /tmp/bar    # Copy /tmp/foo from a remote pod to /tmp/bar locally
+```
+-->
+```bash
+kubectl cp /tmp/foo_dir my-pod:/tmp/bar_dir            # 将 /tmp/foo_dir 本地目录复制到当前命名空间中远程 pod 中的 /tmp/bar_dir
+kubectl cp /tmp/foo my-pod:/tmp/bar -c my-container    # 将 /tmp/foo 本地文件复制到特定容器中远程 pod 中的 /tmp/bar
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:708]:
```suggestion
kubectl cp /tmp/foo my-pod:/tmp/bar -c my-container    # 将 /tmp/foo 本地文件复制到远程 pod 中特定容器的 /tmp/bar
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 7]
```diff
@@ -648,6 +681,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+kubectl cp /tmp/foo my-namespace/my-pod:/tmp/bar       # 将 /tmp/foo 本地文件复制到命名空间 my-namespace 中远程 pod 中的 /tmp/bar
+kubectl cp my-namespace/my-pod:/tmp/foo /tmp/bar       # 将 /tmp/foo 从远程 pod 复制到本地 /tmp/bar
+```
+{{<注>}}
+`kubectl cp` 要求容器映像中存在 'tar' 二进制文件。 如果 'tar' 不存在，`kubectl cp` 将失败。
+对于高级用例，例如符号链接、通配符扩展或文件模式保存，请考虑使用 `kubectl exec`。
+{{</注>}}
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:715]: 同上，不要翻译

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 8]
```diff
@@ -857,15 +928,15 @@ Verbosity | Description
...
+* Read the [kubectl overview](/docs/reference/kubectl/) and learn about [JsonPath](/docs/reference/kubectl/jsonpath).
 
 * See [kubectl](/docs/reference/kubectl/kubectl/) options.
 
 * Also read [kubectl Usage Conventions](/docs/reference/kubectl/conventions/) to understand how to use kubectl in reusable scripts.
 
 * See more community [kubectl cheatsheets](https://github.com/dennyzhang/cheatsheet-kubernetes-A4).
 -->
-* 参阅 [kubectl 概述](/zh/docs/reference/kubectl/overview/)，进一步了解[JsonPath](/zh/docs/reference/kubectl/jsonpath)。
+* 参阅 [kubectl 概述](/zh/docs/reference/kubectl/)，进一步了解[JsonPath](/zh/docs/reference/kubectl/jsonpath)。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:939]:
```suggestion
* 参阅 [kubectl 概述](/zh/docs/reference/kubectl/)，进一步了解 [JsonPath](/zh/docs/reference/kubectl/jsonpath)。
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 9]
```diff
@@ -70,6 +71,19 @@ source <(kubectl completion zsh)  # 在 zsh 中设置当前 shell 的自动补
 echo "[[ $commands[kubectl] ]] && source <(kubectl completion zsh)" >> ~/.zshrc # 在您的 zsh shell 中永久的添加自动补全
 ```
 
+<!--
+### A Note on --all-namespaces
+Appending `--all-namespaces` happens frequently enough where you should be aware of the  shorthand for `--all-namespaces`:
+
+```kubectl -A```
+
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:79]: 这一行不要注释掉再复制粘贴，保留英文原文段落就好。

**Comment by Sea-n** [comment]
There is only 1 newline in this PR.
@zyy19981018 did you force-pushed the wrong commit unintentionally?

**Comment by Sea-n** [comment]
Seems like it's fixed by `reference-12` of umbrella issue.
- #33827

Maybe we could assign ourself to [umbrella issues](https://github.com/kubernetes/website/issues?q=is%3Aissue+zh+Umbrella) next time. 😊
/close

**Pull request closed without merging**